### PR TITLE
[VEN-697] Add skeleton for Swap page

### PR DESCRIPTION
**Pull request by therealemjy**
## Jira ticket(s)

VEN-697

## Changes

- add skeleton for Swap page
- update sidebar so items can display a "NEW" badge

### CODE REVIEW

File: src/components/Layout/Sidebar/index.tsx [hunk 1]
```diff
@@ -60,7 +62,13 @@ export const SidebarUi: React.FC = () => {
                     {t(menuItem.i18nKey)}
                   </Typography>
 
-                  <div className="left-border" />
+                  {menuItem.isNew && (
+                    <div css={styles.listItemNewBadge}>
```
Comment (coreyar) [inline src/components/Layout/Sidebar/index.tsx:66]: Would it make sense to use our chip component here?

File: src/components/Layout/Sidebar/index.tsx [hunk 2]
```diff
@@ -124,6 +132,14 @@ export const SidebarUi: React.FC = () => {
                     >
                       {t(i18nKey)}
                     </Typography>
+
+                    {isNew && (
+                      <div css={styles.listItemNewBadge}>
```
Comment (coreyar) [inline src/components/Layout/Sidebar/index.tsx:137]: Would it make sense to use our chip component here?

**Review by coreyar — APPROVED** [review]

File: src/components/Layout/Sidebar/index.tsx (hunk 1 above)
Comment (therealemjy) [reply src/components/Layout/Sidebar/index.tsx:66]: Unfortunately it doesn't have the same dimensions and styling :/

File: src/components/Layout/Sidebar/index.tsx (hunk 2 above)
Comment (therealemjy) [reply src/components/Layout/Sidebar/index.tsx:137]: Same as above

**Pull request merged**